.github/workflows/ci.yml

```diff
@@ -76,7 +76,6 @@ jobs:
             "test",
             "test_contrast_matrix",
             "test_contrast_list",
-            "test_contrast_matrix",
```

Comment:
Why was that one removed?

Reply:
Because contrast_matrix was tested twice, you can see it already exists in l77 ;)